#!/Users/huohsien/anaconda2/envs/explore_photos_library/bin/python3.12
import sys
from lib2to3.main import main

sys.exit(main("lib2to3.fixes"))
